Requirements for SSE.md

```diff
@@ -119,18 +118,45 @@ TODO: Insert Diagram photo
 
+Community built addons are what drives HA and gives it most of its functionality. HA official supports several addons and allows community users the ability to create their own addons. HA encourages their userbase to create addon repositories (usually hosted on GitHub) and share it with other users for stuff that is not currently supported on HA. While these addons help improve the functionality of HA, it does open the device up for different misuse cases as seen in [Use Case 4](#Use Case 4). Without addons and community support HA will not be able to offer users a robust experience setting HA apart for the other competitors.  
 
 #### [Return to top](#case-list)
 
 ## Planning and Contribution
 
-TODO: Insert repository, project board, and reflection
-Include a link to your team GitHub repository that shows your internal project task assignments and collaborations to finish this task. 
-Include a reflection of your teamwork for this assignment. What issues occurred? How did you resolve them? What did you plan to change moving forward? 
+Each team member was assigned an issue related to 1 of the 5 use/misuse cases. We also used GitHub's [Project Board](https://github.com/Chrs987/HomeAssistant/projects/2) feature and added an "In Review" column so we could track issues that are currently in review. We created separate branches for each team-member and required them to submit a pull-request to the master branch instead of editing the master branch. Each team members pull request required 2 other team members to approve it before it was merged with the master branch. This allowed us to collaborate in the comment section and discuss different ideas on how to approach the assigned use/misuse case. We all met on Monday September 21st via Discord Voice Chat and discussed the potential use cases for the assignment. Mr. Schmitt utilized the screenshare function to give a brief demo of his HomeAssistant setup and overview of the software. The group engaged in a brief discussion and we hypothesized several use cases in preparation for the Tuesday meeting with the professor. We liked the ability to review each other’s changes before pushing it to master and having our own branch. We may change to having each of us "fork" the repository for future assignments, but all agreed that this method worked for now. One of the difficult parts about working on our own branch was parts that we collaborated on like the Planning and Contribution and Observation of Security-Related Configuration and Installation Issues sections. We worked together on these but only created one issue for each part. We plan on brainstorming a better idea for issues that require multiple group members to work on for our next assignment. 
 
 ## Documentation Sources
 
-TODO: Insert documents referenced above (if any)
+* [IFTTT](https://ifttt.com/)
```

Comment:
Can you add these links too, thanks.
[HA Add-on Communication] (https://developers.home-assistant.io/docs/add-ons/communication)
[HA Add-on Security] (https://developers.home-assistant.io/docs/add-ons/security/)
[HA Community Contribution Guidelines] (https://github.com/hassio-addons/repository/blob/master/CONTRIBUTING.md)

Reply:
done